### PR TITLE
[PHILLY 2019] Many updates

### PR DESCRIPTION
**Pull request by peterjshan**
This basically sets up our program, adds speakers, and adds sponsors. A bunch of little things being done here.

@mattstratton This is ready for review now. I don't think I can remove the label. Thanks!

### CODE REVIEW

File: content/events/2019-philadelphia/speakers/katy.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
++++
+Title = "Katy"
```
Comment (bridgetkromhout) [inline content/events/2019-philadelphia/speakers/katy.md]: Are we leaving off her last name by accident or on purpose?

**Comment by bridgetkromhout** [comment]
Hey, @DamionWaltermeyer! I pinged @peterjshan and he said I should chat with you. I'm so excited that we're so close to getting the program up! Let's see if we can clear up these few things before merging...

- I put a commit in to this branch, so please pull it down before force-pushing an override: https://github.com/devopsdays/devopsdays-web/pull/8097/commits/469cbb3dcbd96e0c87e42d8ca4911a5293b29653

- Please reach out to Katy to see if she wants her last name on the website like last year (which we definitely recommend for her professional profile)

- If you look at https://deploy-preview-8097--devopsdays-web.netlify.com/events/2019-philadelphia/program/ you'll see that several speakers from https://deploy-preview-8097--devopsdays-web.netlify.com/events/2019-philadelphia/speakers/ are missing - do you want to add them?

- A number of speakers have their speaker file but not their program file.

File: content/events/2019-philadelphia/speakers/katy.md (hunk 1 above)
Comment (TheKaterTot) [reply content/events/2019-philadelphia/speakers/katy.md]: Hey @bridgetkromhout! I'm not sure why my last name is not on here. I'm not ready to go full Cher yet :D 
Please list my name as Katy Farmer
Thank you for noticing this

**Comment by mattstratton** [comment]
@DamionWaltermeyer please note the comment from @thekatertot that she would like her full name used.

**Comment by peterjshan** [comment]
@bridgetkromhout @mattstratton  - I updated Katy's name. The reason we have extra speakers that don't have a talk is because they are on our waitlist/backup. We are listing them as potential speakers but they may not speak.

**Comment by bridgetkromhout** [comment]
> The reason we have extra speakers that don't have a talk is because they are on our waitlist/backup. We are listing them as potential speakers but they may not speak.

Ah, you should definitely not do that. People end up confused and/or upset (whether it be the speaker or the audience).

**Pull request merged**